tests/fast-integration/telemetry-test/suite.js

```diff
@@ -408,6 +409,50 @@ describe('Telemetry Operator', function() {
         it(`Should delete second TracePipeline '${secondPipelineName}'`, async function() {
           await k8sDelete(secondPipeline);
         });
+
+        const overrideConfig = loadTestData('override-config.yaml');
+        const pipelineName = 'test-trace';
+        it(`Creates a tracepipeline`, async function() {
+          await createTracePipelineInline(pipelineName, 'http://foo-bar');
+          await waitForTracePipeline(pipelineName);
+          await waitForTracePipelineStatusRunning(pipelineName);
+        });
+
+        it('Should have created telemetry-trace-collector secret', async () => {
+          const secret = await getSecret('telemetry-trace-collector', 'kyma-system');
+          assert.equal(secret.data.OTLP_ENDPOINT, 'aHR0cDovL2Zvby1iYXI=');
```

Comment:
Let's maybe base64decode or encode the URL here instead of using a base64 encoded string. It will be more readable